### ISSUE TITLE
Плагин не работает с мультиязычной книгой.

### ISSUE THREAD

**Issue by mathbunnyru** [mikaspell/gitbook-plugin-yametrika#4]
После недавнего обновления плагин всё ещё не работает с мультиязычной книгой. Проблема в том, что в корневой index.html не попадает код для метрики.

Репозиторий - https://github.com/hftbattle/documentation, сайт - https://docs.hftbattle.com.

**Comment by mikaspell** [comment]
в корневой вы имеете ввиду, где предлагается выбор языка? и напишите, какая версия гитбука у вас, плагина и как вы конфиг для плагина прописываете

**Comment by mathbunnyru** [comment]
@mikaspell да, там, где предлагается выбор языка. гитбук версии 3.2.2, плагин последней версии (специально перебилдил книгу, чтоб проверить), конфиг выглядит так (прочитал на npmjs, как надо его писать для 3 версии):
```
    "yametrika": {
      "id": 35257920,
      "webvisor": true,
      "clickmap": true,
      "trackLinks": true,
      "accurateTrackBounce": true,
      "trackHash": true,
      "ut": "noindex"
    }
```

**Comment by mikaspell** [comment]
а на других страницах все ок?

**Comment by mathbunnyru** [comment]
на других страницах в код внедряется Яндекс метрика, но из-за того, что она настроена на главную страницу, она не работает вообще.

**Comment by mikaspell** [comment]
вы специально так настроили?

**Comment by mathbunnyru** [comment]
до этого была одноязычная книга и соотвественно проблем не было, а потом перевели и на английский язык.

**Comment by mikaspell** [comment]
я вчера протестировал, код счетчика корректно добавлялся на все страницы языков, разводящую страницу не смотрел.

По вашему счетчику могут быть тоже вопросы, он же должен все страницы на хосте смотреть, если у него не заданы фильтры, не зависимо от того, на главной есть он или нет

**Comment by mathbunnyru** [comment]
@mikaspell посмотрел ещё раз - вроде работает, спасибо!)  E-mail work@melkee.ru актуальный? Хотелось бы вам подарить книгу за помощь :)

**Comment by mikaspell** [comment]
@mathbunnyru актуальный конечно) не за что) скажите, на разводящей тоже есть?

**Comment by mathbunnyru** [comment]
К сожалению, разводящая страница не работает.

![screen shot 2017-04-24 at 14 54 54](https://cloud.githubusercontent.com/assets/12270691/25336045/5ff67f24-28fe-11e7-8eee-286e95597f3c.png)

**Comment by mikaspell** [comment]
посмотрю сегодня

**Comment by mikaspell** [comment]
разводящую страницу, генерит модуль мультиязычеости, пока не знаю как получить доступ к такой странице, возможно и никак вообще

**Comment by mathbunnyru** [comment]
@mikaspell окей. не забудь посмотреть почту, я тебе прислал e-mail.
